p2p/dandelion/src/tests/router.rs

```diff
@@ -0,0 +1,204 @@
+use std::time::Duration;
+
+use tower::{Service, ServiceExt};
+
+use crate::{DandelionConfig, DandelionRouteReq, DandelionRouterBuilder, Graph, TxState};
+
+use super::*;
+
+#[tokio::test]
+async fn routes_consistent() {
+    // make sure a tx from the same peer goes to the same peer.
+
```

Comment:
Shouldn't this be a doc comment? Same with other tests below.

Reply:
do doc comments make sense for tests?

I'll change these though

Reply:
Shows intention that the comment is describing the whole test and what it does, rather than some code below it.